pipeline-scripts/release.groovy

```diff
@@ -138,7 +142,12 @@ def stageGenPayload(dest_repo, release_name, dest_release_tag, from_release_tag,
         cmd += "--previous \"${previous}\" "
     }
     cmd += "--name ${release_name} "
-    cmd += "--metadata '${metadata}' "
+
+    // Adding metadata will change the payload sha. Ideally, nightlies and
+    // pre-release shas match, so don't add it unnecessarily.
```

Comment:
This seems fine so I'm gonna merge it, but does it accomplish the goal? Won't `oc adm release new` create a new sha regardless? Well, maybe that was in the tl;dr. Let's see.

Reply:
Clayton believed it should be smart enough to preserve the sha if nothing is really changing.